### PR TITLE
Edit Option in result page

### PR DESCRIPTION
**Pull request by pranshugupta54**
- Added Edit option for college predictor and scholarships.
- Updated Scholarship data
- Fixes in Config

### CODE REVIEW

File: pages/api/scholarship-data.js [hunk 1]
```diff
@@ -32,8 +39,13 @@ export default async function handler(req, res) {
         scholarship.Category.length === 0,
       (scholarship) =>
         !familyIncome ||
-        scholarship["Family Income"] === null ||
-        scholarship["Family Income"] <= familyIncome,
+        scholarship["Family Income (in LPA)"] === null ||
+        scholarship["Family Income (in LPA)"] >= familyIncome,
```
Comment (suryabulusu) [inline pages/api/scholarship-data.js:43]: why change the logic here

File: pages/college_predictor.js [hunk 2]
```diff
@@ -29,10 +30,16 @@ const CollegePredictor = () => {
   const [fullData, setFullData] = useState([]);
   const [isLoading, setIsLoading] = useState(true);
   const [error, setError] = useState(null);
+  const [queryObject, setQueryObject] = useState({});
 
+  useEffect(() => {
+    setQueryObject(router.query);
+  }, [router.query]);
+
+  console.log(queryObject);
```
Comment (suryabulusu) [inline pages/college_predictor.js:39]: remove later

**Review by suryabulusu — COMMENTED** [review]
Branch link: https://api-clg.d3gwf1hzebr09y.amplifyapp.com/

Left comments.

Do write the description of PR.

File: pages/api/scholarship-data.js (hunk 1 above)
Comment (pranshugupta54) [reply pages/api/scholarship-data.js:43]: It was wrong previously, 

example: 
 Scholarship requires 4 lakhs income and user entered 10 lakh then not shown

**Review by suryabulusu — APPROVED** [review]

**Pull request merged**